test/nodes/test_agent_rocketride.py

```
@@ -0,0 +1,404 @@
+        """_json_default should convert Decimal-like objects to float."""
+        import decimal
+
+        result = _json_default(decimal.Decimal('3.14'))
+        assert isinstance(result, float)
+        assert abs(result - 3.14) < 0.001
+
+    def test_json_default_datetime(self):
+        """_json_default should convert datetime objects to ISO format."""
+        import datetime
+
+        dt = datetime.datetime(2026, 1, 15, 10, 30, 0)
```

Ruff (0.15.7)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)